platforms/r3-corda/configuration/roles/create/namespace_serviceaccount/tasks/main.yaml

```diff
@@ -3,62 +3,63 @@
 - name: Check if namespace exists
   k8s_facts:
     kind: Namespace
-    name: "{{ component_ns }}"
-    kubeconfig: "{{ kubernetes.config_file }}"
-    context: "{{ kubernetes.context }}"
+    name: "{{ org.name }}-ns"
+    kubeconfig: "{{ org.k8s.config_file }}"
+    context: "{{ org.k8s.context }}"
   register: get_namespace
 
 # Create namespace
-- name: Create namespace for {{ organisation }}
+- name: Create namespace for {{ org.name }}
   include_role:
     name: create/k8_component
   vars:
-    component_name: "{{ component_ns }}"
+    component_name: "{{ org.name }}-ns"
```

Comment:
I think component_ns should be created in the top level, so if someone wants to change the namespace then they need to make minimum changes. Here they will have to change many files. Everything else is fine.

Reply:
I think, the developer should instead search for '-ns' and then check, because at some places we are explicitly doing -ns to get the namespace.